ocp_resources/pipeline.py

```diff
@@ -0,0 +1,55 @@
+# API reference: https://tekton.dev/docs/pipelines/pipelines/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class Pipeline(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        tasks=None,
+        params=None,
+        final_parallel_tasks=None,
+        client=None,
+        yaml_file=None,
+        **kwargs,
+    ):
+        """
+        Args:
+            name (str): Name of the pipeline
+            namespace (str): Namespace of the pipeline
+            tasks (str): Optional: actions to perform in pipeline
```

Comment:
according to https://tekton.dev/docs/pipelines/pipelines/#configuring-a-pipeline
```
[spec](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields) - Specifies the configuration information for this Pipeline object. This must include:

    [tasks](https://tekton.dev/docs/pipelines/pipelines/#adding-tasks-to-the-pipeline) - Specifies the Tasks that comprise the Pipeline and the details of their execution.
```

Reply:
Yes document said it's mandatory but pipeline works without tasks as well. Raised a bug in tekton for it